specification.md

```diff
@@ -775,7 +775,7 @@ complete.
 
 <a name="job-setstatuscallback"></a>
 ```java
-void setStatusCallback(JobStatusCallback? cb)
+void setJobStatusCallback(JobStatusCallback? cb)
```

Comment:
Job.setJobStatusCallback seems a bit redundant, but I agree that it should be consistent.

Reply:
Ok.  Are you suggesting we change both to `setStatusCallback`, or are you OK with this as is?  If the latter, can you resolve this convo and merge?

Reply:
I think we should keep the job one as `setStatusCallback`. I am unsure about the executor one, and I don't think uniformity is important here (despite what I said earlier). But, if you do want uniformity, I'd `setStatusCallback` in both places makes more sense.

Reply:
Ok.  I'll just drop that commit from this PR and we can deal with consistency later if we need to.